RecoLocalTracker/SiPixelClusterizer/plugins/gpuClustering.h

```diff
@@ -35,6 +35,7 @@ namespace gpuClustering {
     }
   }
 
+  template <bool isHIon>
```

Comment:
do you foresee there will be other HIon-related changes needed to this class ?

otherwise, rather than templating on `bool isHIon` it may be more flexible to template directly on the desired `uint32_t maxPixelsPerModule` .

Reply:
No, I don't foresee more HI-related changes, so I followed your suggestion and changed the template argument to uint32_t maxPixelsPerModule.